demo/wdio.conf.js

```diff
@@ -2,7 +2,10 @@
 // import video from 'wdio-video-reporter';
 // But for this demo:
 import video from '../dist/wdio-video-reporter.mjs';
-import path from 'path'
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
```

Comment:
```suggestion
import { fileURLToPath } from 'node:url';
import { dirname } from 'node:path';
```

Reply:
Cool, didn't know it can be done in that way. Thanks a lot. 
Done at [236ec20](https://github.com/webdriverio-community/wdio-video-reporter/pull/98/commits/236ec20f828449f7c46c7bff886f510e4645e0f2)